src/workspaces.c

```diff
@@ -276,6 +276,16 @@ workspaces_switch_to(struct workspace *target, bool update_focus)
 	wlr_scene_node_set_enabled(
 		&server->workspace_current->tree->node, false);
 
+	/* Move Omnipresent views to new workspace */
+	struct view *view;
+	enum lab_view_criteria criteria =
+		LAB_VIEW_CRITERIA_CURRENT_WORKSPACE;
+	for_each_view(view, &server->views, criteria) {
+		if (view->omnipresent) {
+			view_move_to_workspace(view, target);
+		}
+	}
```

Comment:
I think this will currently cause omnipresent views to be always on top of the stacking order when switching workspaces.
But I am not sure how we could prevent that. I am fine with merging this as is for now and think about that part later on.